packages/sitecore-jss-nextjs/src/services/base-graphql-sitemap-service.ts

```diff
@@ -258,14 +262,27 @@ export abstract class BaseGraphQLSitemapService {
     sitePaths.forEach((item) => {
       if (!item) return;
 
-      aggregatedPaths.push(formatPath(item.path));
+      // ItemName-based path
+      const itemNamePath = item.path;
+      aggregatedPaths.push(formatPath(itemNamePath));
+
+      // DisplayName-based path
+      const displayName = item.route?.displayName;
+      if (typeof displayName === 'string' && displayName.trim().length > 0) {
```

Comment:
This will only cover the paths with last part being displayName, right?
I.e. consider this item structure
Home
  Page1 (displayName: Orders)
    Page2 (displayName: Details)
    
This logic will only make aggregated path for /Page1/Details, right?

Reply:
Our implementation covers all nested routes, just like item name based routes.

Reply:
I tested this, works fine.
Next.js will generate that page statically, and as long as
 - The route exists in the sitemap (getStaticPaths)
 - The path matches during navigation
Then it will render successfully.

Reply:
It works fine, but paths with all display names are not generated. From my latest test, here are the pages generated
```
├   ├ /en/_site_base/About/Page-3 (968 ms)
├   ├ /en/_site_base/About/DisplayPage (958 ms)
├   ├ /en/_site_base/DisplayAbout (958 ms)
├   ├ /en/_site_base (958 ms)
├   ├ /en/_site_base/About (957 ms)
```
But there also should be the page for `/en/_site_base/DisplayAbout/DisplayPage`

Reply:
Refactored and all combinations gets pre generated now.